### DISCUSSION TITLE
unexpected crash

### DISCUSSION THREAD

**Discussion by dagosta** [nguyen-group/QERaman#11] [Q&A]
Dear All,

I have a problem with ph_mat.x -- I perform the calculation for Graphene in the example and when the code is writing graphene.elph it crashes after having written a partial file. It seems like the code is trying to access a part of memory that is not allocated properly. This seems to me a racing problem as sometimes the codes complete properly. I have compiled QERaman with intel2023b and 2021b and I see the same problem with both. I have also tried to run the code on a single node or on many nodes with no avail. What could I do?

I cannot attach the files here because they are definitely too big. Here are the last lines of graphene.elph

8        7        4      6.6532      5.7896      111.7800199250    0.5506631675E+02   -0.4994465981E+02    0.2319116634E+02
        8        7        5      6.6532      5.7896      190.7971651906    0.2825997198E-02    0.2784572840E-02    0.4820936226E-03
        8        7        6      6.6532      5.7896      190.7972722422    0.9423265667E-01    0.7785956205E-01   -0.5308184416E-01
        8        8        1      6.6532      6.6532        7.6098015462    0.5577991238E+00   -0.5577991191E+00    0.7288811230E-04
        8        8        2      6.6532      6.6532        8.4043940938    0.4738863048E-01    0.4738767733E-01    0.3005634559E-03
        8        8        3      6.6532      6.6532        8.4072867314    0.7367899399E-02    0.7367862756E-02   -0.2319834894E-04
        8        8        4      6.6532      6.6532      111.7800199250    0.1579336266E+00   -0.1579336265E+00   -0.2212638027E-05
        8        8        5      6.6532      6.6532      190.7971651906    0.3107427966E+02    0.3107427966E+02   -0.4547293922E-04
        8        8        6      6.6532      6.6532      190.7972722422    0.2901286694E+02    0.2901286694E+02    0.1861962995E-04
   0.0000000   0.0000000   0.0000000   0.0000000        0538976304

The last line cannot be correct.

**Comment by nguyen-group** [comment]
Hi Dagosta,

It is not easy for me to check by looking at this. 
Could you give me the input files or could you change the number of k-point in scf.in the file to see what happens?

Best,
Nguyen

**Comment by dagosta** [comment]
Dear Nguyen,I will check with different k-point meshes in the scf.in and let you know. Cheers,Roberto——Roberto D’Agosta, PhDOn 4 Mar 2024, at 22:42, Quantum Materials @ Tohoku University ***@***.***> wrote:﻿
Hi Dagosta,
It is not easy for me to check by looking at this.
Could you give me the input files or could you change the number of k-point in scf.in the file to see what happens?
Best,
Nguyen
